### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by kvendingoldo**

### CODE REVIEW

File: src/python/lean_coffee.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 import sys
 import os
 
-sys.path.append(os.path.join(os.path.dirname(__file__), '..', 'resources'))
+sys.path.append(os.path.join(os.path.dirname(__file__), '../..', 'resources'))
```
Comment (vadim8kiselev) [inline src/python/lean_coffee.py]: Wrong for Windows, you should write like '..', '..'

File: src/python/spreadsheet.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 import sys
 import os
 
-sys.path.append(os.path.join(os.path.dirname(__file__), '..', 'resources'))
+sys.path.append(os.path.join(os.path.dirname(__file__), '../..', 'resources'))
```
Comment (vadim8kiselev) [inline src/python/spreadsheet.py]: Wrong for Windows, you should write like '..', '..'

File: .pylintrc [hunk 3]
```diff
@@ -1,5 +1,6 @@
 [MESSAGES CONTROL]
 # C0111: Missing docstring
 # R0904: Too many public methods
+# R0913: Too-many-arguments
```
Comment (vadim8kiselev) [inline .pylintrc]: Why do you write over dashes?

**Comment by kvendingoldo** [comment]
@vadim8kiselev, On which port does the bot work?

**Pull request closed without merging**